core/src/main/scala/kafka/server/ZkConfigManager.scala

```diff
@@ -38,7 +38,8 @@ object ConfigType {
   val User = "users"
   val Broker = "brokers"
   val Ip = "ips"
-  val all = Seq(Topic, Client, User, Broker, Ip)
+  val ClientMetrics = "client-metrics"
```

Comment:
Given that we don't support client metrics on ZK, I am surprised to see code changes in ZkConfigManager.

Reply:
This is a bit strange I know but ConfigType is defined under this class.